src/app/features/add-new-post/add-new-post.component.ts

```
@@ -0,0 +1,346 @@
+      this.fileInput.nativeElement.value = '';
+    }
+  }
+
+  private updateUploadingState(): void {
+    this.isUploadingAnyImage = this.imageUploadStates.some(state => state.isUploading);
+  }
+
+  onDeleteImage(index: number): void {
+    // Remove from upload states
+    const uploadState = this.imageUploadStates[index];
+    if (uploadState && uploadState.imageId) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)